crates/nu-command/src/strings/str_/replace.rs

```diff
@@ -214,33 +214,33 @@ fn action(
 ) -> Value {
     match input {
         Value::String { val, .. } => {
-            let FindReplace(find, replacement) = FindReplace(find, replace);
+            let FindReplace(f, r) = FindReplace(&find.item, &replace.item);
```

Comment:
I figured it would be less confusing now to send both `find` and `replace` down as `Spanned<String>` instead of one as `Spanned<String>` and another as `String`.

Also, is there any kind of Rust/Nushell style guide that frowns upon single-letter variable names in a narrow context like this one? Would be happy to change if this breaks with convention.

Reply:
We tend to favor more readable which most of the time is more verbose

Reply:
Got it. Will rename and put up again.